src/scim_patch/core.clj

```diff
@@ -101,6 +101,9 @@
 
 (defn op-add
   [schema resource opr]
+  (when (not-any? #(contains? opr %) [:value :path])
```

Comment:
I think `op-replace` has the same bug. Can you fix that too, please?

Reply:
Read [the RFC](https://tools.ietf.org/html/rfc7644#section-3.5.2.1) again and realized that add MUST have a "value" key, which causes other tests to fail. Will need to do additional fixes to the op-replace.